internal/config/config_test.go

```diff
@@ -17,40 +17,206 @@ func TestNewConfig(t *testing.T) {
 		{
 			name: "OK",
 			input: map[string]string{
-				"ADDRESS":          "address",
-				"REST_PORT":        "restport",
-				"JWT_TOKEN_SECRET": "secret",
+				"ADDRESS":            "address",
+				"REST_PORT":          "restport",
+				"JWT_TOKEN_SECRET":   "secret",
+				"GRPC_PORT":          "qw",
```

Comment:
I would suggest using different values for different variables to make sure that they ending up when they supposed to

Reply:
+